MaterialLib/Fluid/Density/LinearTemperatureDependentDensity.h

```diff
@@ -0,0 +1,80 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file:   LinearTemperatureDependentDensity.h
+ * \author: wenqing
+ *
+ * Created on August 10, 2016, 11:34 AM
+ */
+
+#ifndef LINEARTEMPERATUREDEPENDENTDENSITY_H
+#define LINEARTEMPERATUREDEPENDENTDENSITY_H
+
+#include <string>
+
+#include "BaseLib/ConfigTree.h"
+
+#include "MaterialLib/Fluid/FluidProperty.h"
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+/// Linear temperature dependent density model.
+class LinearTemperatureDependentDensity final : public FluidProperty
+{
+public:
+    /// \param config  ConfigTree object which contains the input data
+    ///                including <type>temperature_dependent</type> and it has
+    ///                a tag of <density>
+    explicit LinearTemperatureDependentDensity(BaseLib::ConfigTree const& config)
```

Comment:
I'd suggest to separate parsing of the config and creating an object like it is done e.g. with processes where there is a createABCprocess(config) doing the parsing. Then the constructor assigns the variables. This approach also improves testability.

Suggestion:
 - move parsing into createLinearTemperatureDependentDensity(config)
 - Change constructor to accept three doubles.